tensorboard/plugins/histogram/summary_test.py

```diff
@@ -70,16 +70,6 @@ def test_empty_input_of_high_rank(self):
         buckets = tensor_util.make_ndarray(pb.value[0].tensor)
         np.testing.assert_allclose(buckets, np.array([]).reshape((0, 3)))
 
-    def test_singleton_input(self):
```

Comment:
Ok for a proof of concept, but FWIW, if we remove these test cases from the base test case, we should ensure they get pushed down into all the various test cases - right now this PR only moves them into SummaryV1PbTest and SummaryV2PbTest but that leaves the 3 Op tests uncovered.